### PR TITLE
Fix view prototype value lookup issues

### PR DESCRIPTION
**Pull request by phillipj**
Fixes issue #445 where view object has properties defined in the view's prototype

### CODE REVIEW

**Comment by dasilvacontin** [comment]
I think I found a bug. I'll push some unit tests that break.

**Comment by phillipj** [comment]
Great!

**Comment by dasilvacontin** [comment]
I added a failing test. I suggest rebasing onto `master` to get an improved `AssertionError` diff for the render tests.

**Comment by phillipj** [comment]
Awesome @dasilvacontin, really appreciate your test!

**Comment by phillipj** [comment]
FYI I'm splitting these fixes into two separate PRs, as the first fix context cache fix does not break the test you added

**Comment by dasilvacontin** [comment]
Great! I already merged the other one, thanks! :)

File: test/render-helper.js [hunk 1]
```diff
@@ -26,12 +26,12 @@ function getPartial(testName) {
...
 
 // You can put the name of a specific test to run in the TEST environment
-// variable (e.g. TEST=backslashes vows test/render-test.js)
+// variable (e.g. TEST=backslashes mocha test/render-test.js)
 var testToRun = process.env.TEST;
 
 var testNames;
 if (testToRun) {
-  testNames = [testToRun];
+  testNames = testToRun.split(',');
```
Comment (phillipj) [inline test/render-helper.js]: Small enhancement enabling more than one explicit render test to be runned in isolation

**Comment by phillipj** [comment]
Finally got around to fixing what caused your newly added test to break. Cleaned up the code abit by creating a re-usable method for checking property existence.

**Comment by dasilvacontin** [comment]
It's awesome to have that check as a re-usable method. Awesome, thanks @phillipj!

**Pull request merged**